rfcs/text/0000-rfc-application-layer-bundle-types-and-definitions.md

```diff
@@ -0,0 +1,424 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# Application Layer Bundle Types and Definitions
+
+
+## Summary
+
+This RFC is offered as a refinement and generization of the [RFC: HCA DCP Bundle Types and Definitions]( https://github.com/HumanCellAtlas/dcp-community/pull/93). 
+
+This RFC describes:
+
+1. An application/metadata level mechanism to define bundle types and a simple bundle type hierarchy.
+
+1. An application/metadata level method to specify the type of a bundle.
+ 
+1. Several bundle types designed to match current usage and near term expected usage. 
+
+### Key Differences from RFC: HCA DCP Bundle Types and Definitions
+
+This proposal differs from [RFC: HCA DCP Bundle Types and Definitions](https://github.com/HumanCellAtlas/dcp-community/pull/93/files) mainly in that:
```

Comment:
I see this as a disadvantage.  The number of JSON objects required to actually understand the bundle is already large.  From a client perspective, not including type information in the manifest doesn't seem intuitive.   While the DSS strives to not be tied to just one project, this is an implementation detail we should not inflict on users.  Even if the generic DSS doesn't have a schema associated with bundle.json, the DCP can add one, which the schema link being a DSS configuration option.